README.md

```diff
@@ -1,54 +1,58 @@
 # Researchify
 
-An open source lightweight CMS enabling streamlined webpage development for academic research groups. 
+An open source lightweight CMS enabling streamlined webpage development for academic research groups.
 
 ## Installation
 
 Use the package manager [npm](https://www.npmjs.com/get-npm) to install all required components.
 
 ```bash
-# install api
-cd api
-npm install
-
-# install client
-cd client
```

Comment:
should we be keeping this instruction in case people **want** to run it separately?

Reply:
added

Reply:
cool, thanks